src/account/operations/balance.rs

```diff
@@ -113,7 +114,16 @@ impl AccountHandle {
                                     if let Some(UnlockCondition::StorageDepositReturn(sdr)) =
                                         unlock_conditions.get(StorageDepositReturnUnlockCondition::KIND)
```

Comment:
FYI you can use `.storage_deposit_return()`, it's shorter

Reply:
Thanks, will update in a new PR